samples/cellular/nrf_cloud_multi_service/overlay_nrf7002ek_wifi_no_lte.conf

```diff
@@ -31,12 +31,49 @@ CONFIG_BOOTLOADER_MCUBOOT=n
 CONFIG_IMG_MANAGER=n
 CONFIG_MCUBOOT_IMG_MANAGER=n
 
+## Enable TFM and Trusted Execution so that we can use Protected Storage for credentials storage.
+## Please note that even when using Protected Storage, credentials are still retrieved into insecure
+## memory when in use.
+CONFIG_BUILD_WITH_TFM=y
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
+CONFIG_TFM_BUILD_NS=y
+
+## Enable Protected Storage
+CONFIG_TFM_PARTITION_PROTECTED_STORAGE=y
+
+## Configure TFM Profile. The NOT_SET profile will enable all features.
```

Comment:
```suggestion
## Configure TFM Profile. The NOT_SET profile will disable all features. 
```

Reply:
Is this true? I thought the NOT_SET profile enables a bunch of features that are not in the TFM_MINIMAL profile?